From 1a6c819da07f3900b7dd185d1bc44fa9315ccaa7 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Sun, 4 Jan 2026 23:54:58 +0100
Subject: [PATCH] arm64: dts: rockchip: rk3399-pinephone-pro: Disable interrupt
 GPIO for WiFi for the Pinephone Pro

This breaks at least dev version of Pinephone Pro. Or use of Pinephone
Pro
with firmware configurations that don't use OOB interrupts.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
index 4428b0f25e1a..bb72e41366dc 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
@@ -1243,9 +1243,9 @@ &sdio0 {
 	brcmf: wifi@1 {
 		compatible = "brcm,bcm4329-fmac";
 		reg = <1>;
-		interrupt-parent = <&gpio4>;
-		interrupts = <RK_PD0 IRQ_TYPE_LEVEL_HIGH>;
-		interrupt-names = "host-wake";
+		//interrupt-parent = <&gpio4>;
+		//interrupts = <RK_PD0 IRQ_TYPE_LEVEL_HIGH>;
+		//interrupt-names = "host-wake";
 		pinctrl-names = "default";
 		pinctrl-0 = <&wifi_host_wake_l>;
 	};
-- 
2.55.0

